src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -227,5 +231,24 @@ private async Task GenerateDeploymentRecipeSnapShot(Recommendation recommendatio
             var recipeSnapshotBody = JsonConvert.SerializeObject(recipe, Formatting.Indented);
             await _fileManager.WriteAllTextAsync(recipeSnapshotFilePath, recipeSnapshotBody);
         }
+
+        /// <summary>
+        /// This method takes the full path of the directory at which the CDK deployment project is saved and adds a
+        /// corresponding entry in the deployment-manifest file.
+        /// </summary>
+        /// <param name="saveCdkDirectoryFullPath">The full path of the directory at which the CDK deployment project is saved</param>
+        /// <returns></returns>
+        private async Task UpdateDeploymentManifestFile(string saveCdkDirectoryFullPath)
```

Comment:
nit: I would prefer using AbosolutePath and RelativePath terms.

Reply:
I have used `xyzFullPath` for **all** variables that refer to the absolute path. I can put a a separate PR to fix this.